charts/alfresco-repository/docs/keystores.md

```diff
@@ -0,0 +1,83 @@
+# Using custom keystore
+
+Alfresco uses a set of keystores for different purposes. Among others thing
+keystores can be used for metadata encryption or traffic encryption.
+
+The alfresco-content-repository image comes with a keystore which contains a
+single security key. That key is primarily used for metadata encryption and it
+required when the repository starts up. In production environment it is
+recommended to use a custom keystore rather than the one present in the image.
+
+One may also need to have additional keystores deployed, for instance to allow
+using TLS encryption for a service the repository provides using an x509
+certificate. We describe below how to do both.
+
+## The metadata keystore
+
+This keystore is used for property encryption and needs to remain identical
+throughout the life of the repository. Losing this keystore or its passwords
+will result in a repository that cannot startup so it is absolutely crucial to
+store these assets in a version control system like git (both the keystore and
+its password).
```

Comment:
I would not suggest git to store credentials, just say that is important to properly back it up together with data otherwise it will be unusable.